tests/utils_tests/test_html.py

```diff
@@ -422,6 +425,80 @@ def test_urlize(self):
             with self.subTest(value=value):
                 self.assertEqual(urlize(value), output)
 
+    def test_urlize_assume_https(self):
```

Comment:
I think only the 1st and 3rd test case is necessary
We try not to add double cover in our test suite